### ISSUE TITLE
Warning color not accessbile

### ISSUE THREAD

**Issue by DusanMilko** [carbon-design-system/carbon-elements#516]
For displaying errors the color for text is a red - #da1e28;
This red passes most accessibility on white background.

For displaying warnings the color for text is a yellow - #fdd13a;
This yellow **fails** all accessibility on white background.

Recommend using a yellow that passes on white background for atleast AA standard.

Also minor suggestion to updating the red to pass all accessibility.

**Comment by dakahn** [comment]
@DusanMilko Hey! Thanks for talking about accessibility and color contrast. Couple questions: what are you using to test color contrast and what contrast rules are you testing against?

**Comment by DusanMilko** [comment]
@dakahn I used various tools like google chromes color contrast checker in the dev tools, or https://webaim.org/resources/contrastchecker/ or other sources online.
https://developers.google.com/web/updates/2018/01/devtools#contrast

Checking both AA and AAA standards as mentioned above.

**Comment by dakahn** [comment]
@carbon-design-system/design
